plugins/html/icon_html/actions/epub/action.py

```diff
@@ -1,10 +1,10 @@
 import insightconnect_plugin_runtime
 import base64
-
-from insightconnect_plugin_runtime.exceptions import PluginException
 import pypandoc
 import re
-from .schema import EpubInput, EpubOutput
+
+from insightconnect_plugin_runtime.exceptions import PluginException
+from .schema import EpubInput, EpubOutput, Input, Output
```

Comment:
```suggestion
from .schema import EpubInput, EpubOutput, Input, Output, Component
```
then on line 13 same again, change description to `Component.DESCRIPTION`